src/official-widgets/embedded-search-results/embedded-search-results.spec.tsx

```diff
@@ -641,4 +647,29 @@ describe('embedded-search-result', () => {
       expect(productCardImage.getAttribute('src')).toEqual(`https://main-image-${scrambledOrder[idx] + 1}`);
     });
   });
+
+  it('should show image results successfully with cropped view', () => {
```

Comment:
Again, there shouldn't be a need for separate test case. This is essentially the same as the test case `should render successfully with im-url`.